### DISCUSSION TITLE
Install docker in my doodba project?

### DISCUSSION THREAD

**Discussion by ygol** [Tecnativa/doodba-copier-template#442] [Q&A]
Hi,
I need to install docker inside my doodba image. should I add a script in entrypoint.d ? 
would adding the following in a file there is the way?
> curl -sSL https://get.docker.com/ | sh

Is there another easier way?
TIA

**Comment by pedrobaeza** [comment]
You should check in general Docker forums, as this is not something specific of Doodba itself. Anyway, I think it's better to communicate several Docker containers instead having to do "Docker in docker" (DiD).

**Reply by ygol** [reply]
thank you @pedrobaeza 
I am trying to figure how I install it in doodba. 
Where should I add a script that will be executed when building the image. 
In this case I can't add it to one of the dependencies (pip, apt, etc). I tried adding a file under the "build.d" directory but it doesn't seem to work.

**Reply by chienandalu** [reply]
>  I tried adding a file under the "build.d" directory but it doesn't seem to work.

Did you give it execution permissions? (i.e.: `chmod +x`)

Anyway, if you just need additional docker images, just add them as a service in your docker-compose.yaml files (test,prod,devel,common)
